src/melee/pl/types.h

```diff
@@ -4,6 +4,8 @@
 #include <platform.h>
 #include <placeholder.h>
 
+#include "forward.h"
```

Comment:
One quirk of our compiler is that relative includes don't work properly, they're always relative to the .c file being compiled. So for example, if this gets included from melee/ft/fighter.c, it will actually include melee/ft/forward.h instead of melee/pl/player.h. I'd suggest fully qualifying this as <melee/pl/forward.h> instead. Not a big deal here but something to keep in mind.

Reply:
It's fine to use relative includes in C files, just not in headers.